analyses/immune-deconv/02-visualize_quantiseq.Rmd

```diff
@@ -240,9 +251,29 @@ subtypes_celltypes_plot <- ggplot(quantiseq_no_ratio) +
         axis.line = element_line(size = rel(0.7)),
         axis.ticks = element_line(size = rel(0.7)))
 
-subtypes_celltypes_plot
+# Add legend
+cg_color_df <- quantiseq_no_ratio %>%
+  select(cancer_group_hex, cancer_group_display) %>%
+  distinct()
+
+cg_color_list <- cg_color_df$cancer_group_hex
+names(cg_color_list) <- cg_color_df$cancer_group_display
 
-ggsave(subtypes_celltypes_plot_file, subtypes_celltypes_plot, width = 13, height = 4)
+legend_plot <- ggplot(quantiseq_no_ratio) + 
+  aes(x = score, y = score, color = cancer_group_display) + 
+  geom_point() +
+  scale_color_manual(name = "Cancer group", values = cg_color_list) + 
+  theme_pubr()
```

Comment:
I'm realizing this legend is way to big in the output! Fix like this (or maybe sizes 5 and 7 instead of 6/8? whatever you think looks better!) - 

```suggestion
  theme_pubr() + 
    # Need to make text very small
  theme(
    legend.text = element_text(size = 6),
    legend.title = element_text(size = 8)
  )
```